pkgs/by-name/xr/xrsh/package.nix

```
@@ -0,0 +1,65 @@
+{
+  stdenv,
+  fetchFromGitea,
+  makeWrapper,
+  writeShellApplication,
+  python3Packages,
+  lib,
+  unstableGitUpdater,
+}:
+
+stdenv.mkDerivation rec {
```

`rec` is not needed here.

```suggestion
stdenv.mkDerivation {
```